airbyte-workers/src/main/java/io/airbyte/workers/process/KubePodProcess.java

```diff
@@ -158,7 +158,7 @@ private static Container getInit(boolean usesStdin, List<VolumeMount> mainVolume
       initEntrypointStr = String.format("mkfifo %s && ", STDIN_PIPE_FILE) + initEntrypointStr;
     }
 
-    initEntrypointStr = initEntrypointStr + String.format(" && until [ -f %s ]; do sleep 5; done;", SUCCESS_FILE_NAME);
+    initEntrypointStr = initEntrypointStr + String.format(" && until [ -f %s ]; do sleep 0.00001; done;", SUCCESS_FILE_NAME);
```

Comment:
why do we need the sleep at all? can we just remove the `until` loop?

Reply:
We need to wait until the final file is uploaded. Is there a better way to do that than an `until` loop in bash?

I'm going to merge as is and will follow up on this if you have a suggestion.

Reply:
I think you're right. I was thinking about it wrong. This does not disprove that there could be a better way of doing it, but I don't know it 😛